src/js/components/Anchor/Anchor.js

```diff
@@ -75,6 +75,11 @@ const Anchor = forwardRef(
     const first = reverse ? label : coloredIcon;
     const second = reverse ? coloredIcon : label;
 
+    const anchorChildCustomStyle = {
```

Comment:
Instead of using the `style` property, we should be creating a variant of Box for this:
```
import styled from 'styled-components';
...
const InlineBox = styled(Box)`
  display: inline-flex;
  vertical-align: bottom;
`;
...
```

Reply:
Thanks, @ericsoderberghp I will refactor it.